packages/react-components/src/organisms/TeamCreateOutputFormSharingCard.tsx

```diff
@@ -142,6 +147,21 @@ const TeamCreateOutputFormSharingCard: React.FC<TeamCreateOutputFormSharingCardP
           value={sharingStatus}
           onChange={onChangeSharingStatus}
         />
+        {sharingStatus === 'Public' ? (
+          <LabeledDateField
+            title={'Date Published'}
+            subtitle={'(optional)'}
+            description={
+              'This should be the date your output was shared publicly on it’s repository.'
+            }
+            onChange={onChangePublishDate}
+            value={publishDate}
+            max={new Date()}
+            getValidationMessage={() =>
+              'Publish date cannot be greater than today'
```

Comment:
Can the user somehow enter other invalid data? In that case the validation message would be wrong

Reply:
I don't think so, the input type is date